.storybook/mock-content/linksBar.js

```diff
@@ -6,37 +6,37 @@ export const linksBarMock = (query) => {
 			children: [
 				{
 					_id: "id_1",
-					name: "test link 1",
```

Comment:
not sure I totally follow why this mock content was changed. Is something wrong with the existing text? I usually keep what's there so that it's easier to tell what's changed